### PR TITLE
Adds the N.E.R.F gun

### PR DESCRIPTION
**Pull request by GuppyLaxx**
This pull request adds a new and exclusive N.E.R.F gun, which, when shot at something, makes something "cool" and" good" into something utterly pointless. Why must you add this to the game? Well, doing calculations which took my entire day, I've come to the conclusion that if this was added, I would be helping my buddy Poojawa by reducing their time spent coding by 99%, so they could instead put this time to a better use.
Now when my buddy, Poojawa, just wants to do some **GOOD** for the community, they can just hop on,  pull this magical tool out of their backpack, and promptly aim then shoot it at whatever object they wish to make objectively worse. Of course, such a powerful tool requires some work, even with its advanced technology. If the gun seemingly doesn't work, do not worry, but instead COCK the gun, and you'll know when that puppy's ready!
Thank you for coming to my TED talk.

### CODE REVIEW

**Comment by chaoko99** [comment]
http://www.scp-wiki.net/scp-3108

**Comment by peeesh** [comment]
a hero.

**Comment by Poojawa** [comment]
I'm flattered to have a troll PR, but, no thanks.

**Comment by chaoko99** [comment]
Troll PRs tend not to suggest effort in them.

**Comment by chaoko99** [comment]
ALSO PUSH UPSTREAM DANGIT

File: modular_citadel/code/modules/projectiles/guns/nerf.dm [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+		charges++
+		recharge_newshot()
+		playsound(src, 'modular_citadel/sound/weapons/itsnerfornothing.ogg', 75)
+	else
+		to_chat(user, "<span class='warning'>The [src] locks up, it's already charged.</span>")
+		playsound(src, 'sound/weapons/gun_dry_fire_1.ogg', 75)
+
+/obj/item/ammo_casing/nerf/
+	projectile_type = /obj/item/projectile/nerf
+	harmful = FALSE // nerfing surely cant hurt you
```
Comment (Useroth) [inline modular_citadel/code/modules/projectiles/guns/nerf.dm:23]: My favourite line,  I think

**Comment by silicons** [comment]
sorry this is a meme pr even if it's hilarious as fuck :^(

**Pull request closed without merging**